policies/aws_account_policies/aws_resource_required_tags.py

```diff
@@ -1,19 +1,24 @@
+import ast
+from unittest.mock import MagicMock
+
 # REQUIRED_TAGS_MAPPINGS maps resource types to a set of tag keys required for that resource.
-# Example: REQUIRED_TAGS_MAPPINGS = {'AWS.EC2.Instance.Snapshot': {'TagOne', 'TagTwo'}}
-# The above example would check all EC2 instances for the presence of tags keyed TagOne and TagTwo
+# Example: REQUIRED_TAGS_MAPPINGS = {'AWS.EC2.Instance.Snapshot': {'Owner', 'CreatedBy'}}
+# The above example would check all EC2 instances for the presence of tags keyed Owner and CreatedBy
+
 REQUIRED_TAGS_MAPPINGS = {}
 
 
 def policy(resource):
-    if resource["Tags"] is None:
-        return True
-
-    # Check if this resource type has any tag requirements.
-    # If this resource type doesn't have tag requirements, the policy should be updated.
-    if resource["ResourceType"] not in REQUIRED_TAGS_MAPPINGS:
-        return False
-
-    required_tags = REQUIRED_TAGS_MAPPINGS[resource["ResourceType"]]
-    actual_tags = set(tag["Key"] for tag in resource["Tags"])
-
-    return required_tags.issubset(actual_tags)
+    # pylint: disable=not-callable
+    # pylint: disable=global-statement
+    global REQUIRED_TAGS_MAPPINGS
+    if isinstance(REQUIRED_TAGS_MAPPINGS, MagicMock):
+        REQUIRED_TAGS_MAPPINGS = ast.literal_eval(REQUIRED_TAGS_MAPPINGS())
```

Comment:
in other areas, we're using `json.loads()` because of the volatility of `ast.literal_eval`

Reply:
the expected value isn't json (`{KeyOne, KeyTwo}`) so json was complaining.
I could change it to expect a list instead if we don't want to use ast

Reply:
ah, I had forgotten about the set thing. I'm ok with either a list or we handle the set situation like we do in [gsuite_drive_visibility_change](https://github.com/panther-labs/panther-analysis/blob/master/rules/gsuite_reports_rules/gsuite_drive_visibility_change.py#L59-L60)

Reply:
updated to list